File: test-scripts/clean_311.py
#!/Users/neeraj/.miniconda3/envs/py35/bin/python
import sys

g = open(sys.argv[1], 'w')
numcol = 0
while True:
    line = sys.stdin.buffer.readline().decode('utf-8', errors = 'ignore')
    if not line: break
    if numcol == 0 and line.count("|") > 0:
        numcol = line.count("|")
        g.write(line)
    elif numcol ==0:
        g.write(line)
    else:
        g.write("|".join(line.split("|")[:numcol+1]) + ";".join(line.split("|")[numcol+1:]))
g.close()

File: test-scripts/clean_311.js
#!/usr/bin/env node

const fs = require('fs');
const readline = require('readline');

[nodeLocation, commandFile, outPath] = process.argv;

let outStream = fs.openSync(outPath, 'w');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
  terminal: false
});

let columnCount = 0;

rl.on('line', function(line) {
  let splitRow = line.split('|');
  let writeLine = line;
  if (columnCount == 0) {
    if (splitRow.length > 1) {
      columnCount = splitRow.length - 1;
    }
  } else {
    let comments = splitRow.splice(columnCount, splitRow.length - columnCount);
    splitRow.push(comments.join('; '));
    writeLine = splitRow.join('|');
  }

  fs.write(outStream, writeLine + '\n');
});
